addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -759,16 +759,16 @@ public Type toType(Datapoint datapoint, byte[] data) {
                 return null;
             }
 
+            logger.trace("value of DPT = '{}'", value);
             if (typeClass.equals(PercentType.class)) {
-                return PercentType.valueOf(value.split(" ")[0]);
+                return PercentType.valueOf(value.split(" ")[0].replace(",", "."));
```

Comment:
Is there any particular reason why bothering with the serialized form at all? Shouldn't `translator.getNumericValue()` give us exactly the numeric value that we need?